console/src/components/CompactHVAC/CompactHVAC.js

```diff
@@ -0,0 +1,56 @@
+import Picker                from '@enact/agate/Picker';
```

Comment:
Please update this and the following lines to not use extra spacing before the `from` keyword.